apps/condo/pages/employee/index.tsx

```diff
@@ -180,20 +180,21 @@ const TicketsPage = () => {
                                     </Col>
                                 </Row>
                         }
-                    </OrganizationRequired>
-                </PageContent>
+                    </PageContent>
+                </OrganizationRequired>
             </PageWrapper>
         </>
     )
 }
 
 const HeaderAction = () => {
     const intl = useIntl()
+    const BackButtonLabel = intl.formatMessage({ id: 'pages.condo.employee.PageTitle' })
```

Comment:
Is the `UpperCamelCase` style required here? Why not `backButtonLabel`? Just pointing out. If done intentionally - don't listen to me :-)